components/ServiceCard/index.js

```diff
@@ -0,0 +1,38 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import useSWR from "swr";
+
+export default function ServiceCard() {
+  const router = useRouter();
+  const { id } = router.query;
+  const { data, isLoading, error } = useSWR(`/api/services/${id}`);
+
+  if (isLoading) {
+    return <h2>Loading...</h2>;
+  }
+
+  if (error) {
+    return;
+  }
+
+  return (
+    <>
+      <h2>Service</h2>
+      <article>
+        <Image
+          src={data.image}
+          width={100}
+          height={100}
+          alt="picture of the service"
+        />
+        <h3>{data.name}</h3>
+        <p>{data.description}</p>
+        <p>Date: {data.date}</p>
+        <p>Time: {data.time}</p>
+        <p>Price: {data.price} €</p>
```

Comment:
Utilizing semantic HTML elements can provide both clarity to developers and improved accessibility for users. Using appropriate semantic elements also makes your HTML more meaningful to both browsers and search engines, potentially improving SEO. 🤓

```suggestion
        <dl>
          <dt>Date:</dt>
          <dd>{data. date}</dd>
          
          <dt>Time:</dt>
          <dd>{data.time}</dd>

          <dt>Price:</dt>
          <dd>{data.price} €</dd>
        </dl>
```